dev-qt/prison/prison-5.38.recipe

```diff
@@ -0,0 +1,49 @@
+SUMMARY="A barcode api"
+DESCRIPTION="prison has a prison::AbstractBarcode, which is the base class  \
+for the actual barcode generators, currently prison::QRCodeBarcode and \
+prison::DataMatrixBarcode is the two implemented barcode generators."
```

Comment:
Just because the author used bad grammar doesn't mean we have to copy it word for word.  This should be "are" instead of "is"